src/components/home/AllCollections.js

```diff
@@ -145,10 +145,14 @@ function AllCollections() {
         ?.id,
     )
     ?.sort(compare);
+  const languageList = allCategories?.get(
+    categoryTypes?.find((type) => type.name === CATEGORY_TYPES.LANGUAGE)?.id,
+  );
 
   // state variable to record selected options
   const [selectedLevel, setSelectedLevel] = useState([]);
   const [selectedDiscipline, setSelectedDiscipline] = useState([]);
+  const [selectedLanguage, setSelectedLangauge] = useState([]);
```

Comment:
We can use plurals for these arrays since we can choose multiple choices, right?